fastlane/metadata/he/keywords.txt

```diff
@@ -1 +1 @@
-רשתות חברתיות,פתקים,Jetpack,כתיבה,תיוג גיאוגרפי,מדיה,בלוג,אתר,כתיבת בלוג,כתיבת יומן
\ No newline at end of file
+blogger,writing,blogging,web,maker,online,store,business,make,create,write,blogs
```

Comment:
🤔 

Looks like the GlotPress translation was... English? 

<img width="958" alt="image" src="https://user-images.githubusercontent.com/1218433/158925662-f98d5eb8-e013-4b0a-994c-66faff0de14a.png">